services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java

```diff
@@ -21,19 +21,33 @@
 package app.coronawarn.server.services.distribution.config;
 
 import app.coronawarn.server.common.protocols.external.exposurenotification.SignatureInfo;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Pattern;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
 
 @Component
 @ConfigurationProperties(prefix = "services.distribution")
+@Validated
 public class DistributionServiceConfig {
 
   private Paths paths;
   private TestData testData;
+  @Min(7)
+  @Max(28)
   private Integer retentionDays;
+  @Min(120)
+  @Max(720)
   private Integer expiryPolicyMinutes;
```

Comment:
Could also be 0 (e.g. in `demo` profile).

Reply:
Ok, but 120m is the minimum defined in the Google / Apple Specs, thats why I thought keeping it at that might be good idea.

Reply:
so shall we put the min value to 0 then? Or leave it at 120?

Reply:
@pithumke @UnchartedBull Did you come to an agreement regarding this already?

Reply:
Should be fine with `120`. Looks like the `demo` profile doesn't change this property after all.